plugins-bundled/internal/plugin-admin-app/src/pages/nav.ts

```diff
@@ -0,0 +1,67 @@
+import { NavModel, NavModelItem } from '@grafana/data';
+
+export enum CatalogTab {
+  Browse = 'browse',
+  Discover = 'discover',
+  Library = 'library',
+}
+
+export function getCatalogNavModel(tab: CatalogTab, baseURL: string): NavModel {
+  const pages: NavModelItem[] = [];
+
+  if (!baseURL.endsWith('/')) {
+    baseURL += '/';
+  }
+
+  pages.push({
+    text: 'Browse',
+    icon: 'icon-gf icon-gf-apps',
+    url: `${baseURL}`,
+    id: CatalogTab.Browse,
+  });
+
+  //   pages.push({
```

Comment:
The discover route definition and component are still in the codebase but doesn't look like we can navigate to it anymore?

Reply:
I was reluctant to remove it since we will likely want something like it, and it is non-trivial to implement -- but it seems to awkward to include in this initial version